cosmwasm/wasm-wiring/ante.txt

```diff
@@ -0,0 +1,64 @@
+package app
```

Comment:
You can use `.plush` template files like the ignite. We have some helper packages for it. See: https://github.com/ignite/cli/tree/ca31200f375f08c97b2c49a029a47523d400d821/ignite/pkg/xgenny